src/extension/background-script/actions/blocklist/get.ts

```diff
@@ -0,0 +1,21 @@
+import { MessageBlocklistDelete } from "~/types";
+
+import db from "../../db";
+
+const get = async (
+  message: MessageBlocklistDelete
```

Comment:
Wrong type? This would be noticed with unit tests later on :)

Reply:
I didn't know if I should create a new `MessageBlocklistGet` type that would be same as `MessageBlocklistDelete`. is that the best practice? I felt like it would be repetitive.

Reply:
The `action` would be different.  
We need this anyways if we add tests later because. See `account`-types. Leaving this would be confusing.  

We would also need a type here later: https://github.com/getAlby/lightning-browser-extension/pull/825/files#diff-fd1daf979191de8eaf96a501e8ef14d98ca457e51adc21c9fc1ed4110271c997R4
For now it's ok to leave it out.

If things get too repetitive later we can optimize the types. I believe we need to create some sort of overview and testing first and optimize based on that.

Reply:
good point on `action`. done.